_posts/2024-04-15-trino-fest-2024-approaches.md

```diff
@@ -0,0 +1,124 @@
+---
+layout: post
+title: "A sneak peek of Trino Fest 2024"
+author: "Cole Bowden"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-fest-2024/announcement-banner.png
+show_pagenav: true
+---
+
+Trino Fest is drawing ever closer, and Commander Bun Bun has been hard at work
+behind the scenes arranging the schedule and making sure that Trino's trip to
+Boston is going to be a great one. In case you missed it,
+[we announced Trino Fest]({{site.baseurl}}{% post_url 2024-02-20-announcing-trino-fest-2024 %})
+a couple months ago, and if you have missed it, make sure to go register to
+attend! All our speakers will be in person in downtown Boston on the 13th of
+June, with plenty of opportunities for networking and a 2-hour happy hour at the
+end of the day. But if you can't make the trip up to enjoy the lovely New
+England summer, we'll also be livestreaming the event, and you can register to
+join us virtually.
+
+<div class="card-deck spacer-30">
+    <a class="btn btn-orange" href="http://www.starburst.io/info/trino-fest-2024?utm_medium=trino&utm_source=website&utm_campaign=Global-FY25-Q2-EV-Trino-Fest-2024&utm_content=Blog-2">
+        Register to attend!
+    </a>
+</div>
+<div class="spacer-30"></div>
+
+Still on the fence, though? Read on for a sneak peek at our speaker lineup and
+previews of their talks. Companies being represented include names like Apple 
+and Amazon, so you know it's going to be good.
+
+<!--more-->
+
+## Data lakes
+
+It's no secret that using Trino as part of your data lake has become one of its
```

Comment:
well... it was the major use case from the start .. so either reframe to lakehouses .. or change altogether